From e2a9bf836465dbb138ee83ccbe41b385e4648014 Mon Sep 17 00:00:00 2001
From: Igor Murzov <igor@gplsoft.org>
Date: Sun, 18 Apr 2010 22:02:45 +0400
Subject: [PATCH 2/6] fix spurious path to rpm file with -n option

there is no need to prepend $CWD to the path to file,
as it produce wrong path in the case, when path to file
is not relative to the current working directory
---
 rpm2tgz |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/rpm2tgz b/rpm2tgz
index f5f206c..c41f456 100755
--- a/rpm2tgz
+++ b/rpm2tgz
@@ -213,11 +213,12 @@ for i in $* ; do
   # makepkg to produce the .tgz:
   if [ "$(basename $0)" = "rpm2tgz" ]; then
     ( 
-      cd $TMPDIR
       if [ "$META_NAME" = "true" ]; then
-        get_meta_data $CWD/$i
+        get_meta_data $i
+        cd $TMPDIR
         makepkg -l y -c n $CWD/$PRGNAM-$VERSION-$ARCH-${BUILD}.tgz
       else
+        cd $TMPDIR
         makepkg -l y -c n $CWD/$(basename $i .rpm).tgz
       fi
     )
-- 
1.7.0.2

